swagger.json

```diff
@@ -2386,18 +2386,6 @@
                 "destination_lng"
             ],
             "properties": {
-                "driver_id": {
```

Comment:
Can we also update route creation/update endpoints saying only one of `driver_id` or `vehicle_id` should be populated?
Right now, the wording is slightly misleading and could be interpreted as it's ok to populate both `driver_id` and `vehicle_id`.

>  vehicle_id: ID of the vehicle assigned to the dispatch route. One of driver_id OR vehicle_id is required.
> driver_id: ID of the driver assigned to the dispatch route. One of driver_id OR vehicle_id is required.

Reply:
Sure, updated.